tests/test_fairstep.py

```diff
@@ -74,13 +76,21 @@ def test_construction_from_rdf_remove_irrelevant_triples(self):
         """
         rdf = read_rdf_test_resource('sample_fairstep_nanopub.trig')
         uri = 'http://purl.org/np/RACLlhNijmCk4AX_2PuoBPHKfY1T6jieGaUPVFv-fWCAg#step'
-        test_namespace = rdflib.Namespace('http://example.com#')
+        test_namespace = rdflib.Namespace(
+            'http://purl.org/np/RACLlhNijmCk4AX_2PuoBPHKfY1T6jieGaUPVFv-fWCAg#')
         test_irrelevant_triples = [
             # A random test statement that has nothing to do with this step
             (test_namespace.test, test_namespace.test, test_namespace.test),
             # A precedes relation with another step that is part of the workflow RDF, not this
             # step RDF.
-            (rdflib.URIRef(uri), namespaces.DUL.precedes, test_namespace.other_step)
+            (rdflib.URIRef(uri), namespaces.DUL.precedes, test_namespace.other_step),
+            # A triple about a different step that is also a manual task
+            # TODO: This fails for 2 reasons:
```

Comment:
This still needs a fix, awaiting an answer on: https://stackoverflow.com/questions/65493413/sparql-arbitrary-length-property-path-query-with-arbitrary-but-directed-properti